app/test/integration_test.go

```diff
@@ -385,66 +308,6 @@ func (s *IntegrationTestSuite) TestEmptyBlock() {
 	}
 }
 
-// TestSubmitPayForBlob_blobSizes verifies the tx response ABCI code when
-// SubmitPayForBlob is invoked with different blob sizes.
-func (s *IntegrationTestSuite) TestSubmitPayForBlob_blobSizes() {
```

Comment:
[question] was this test moved or just deleted? It doesn't look like the original PR deleted them: https://github.com/celestiaorg/celestia-app/pull/2455/files#diff-d66f948da147c1b292dcf8131bd525c02c6e23daa6c64712a85e53476271cc65L60

Reply:
ditto